server/nuxeo-nxr-server/src/main/resources/templates/common-base/client/scripts/taskGC.groovy

```diff
@@ -0,0 +1,33 @@
+import org.apache.logging.log4j.LogManager
+import org.nuxeo.ecm.platform.task.Task
+import org.nuxeo.ecm.core.api.IdRef
+import org.nuxeo.ecm.core.api.ScrollResult
+import org.nuxeo.runtime.transaction.TransactionHelper
+import java.util.HashMap
+
+log = LogManager.getLogger("taskGC.groovy")
```

Comment:
Even if it's not mandatory, could you take the habit to declare your vars at least with `def` if you don't want to type them?

```suggestion
def log = LogManager.getLogger("taskGC.groovy")
```

Same for all vars in the script.